spec/rails_helper.rb

```diff
@@ -1,7 +1,15 @@
 # frozen_string_literal: true
 
-require 'simplecov'
 require 'spec_helper'
+require 'simplecov'
+
+SimpleCov.start 'rails' do
+  add_filter 'app/channels/application_cable/'
+  add_filter 'app/jobs/'
+  add_filter 'app/mailers/'
+  add_filter 'app/models/application_record.rb'
+  minimum_coverage 95
+end
```

Comment:
Move this code to `.simplecov` file at the root directory of project
You can check it [here](https://github.com/simplecov-ruby/simplecov#using-simplecov-for-centralized-config)